common/fixation_entry.go

```diff
@@ -116,8 +115,9 @@ func (fs *FixationStore) AppendEntry(
 ) error {
 	safeIndex, err := types.SanitizeIndex(index)
 	if err != nil {
-		details := map[string]string{"index": index}
-		return utils.LavaError(ctx, ctx.Logger(), "AppendEntry_invalid_index", details, "invalid non-ascii entry")
+		return utils.LavaFormatError("invalid non-ascii entry", err,
```

Comment:
description string is a duplicate in all of the functions, change it to give us an idea where it came from